com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -805,5 +802,36 @@ bool CanCopyDepth(ref CameraData cameraData)
             bool msaaDepthResolve = false;
             return supportsDepthCopy || msaaDepthResolve;
         }
+
+        /// <summary>
+        /// Recreate post process passes. If renderer already contains valid post process passes, they will be replaced by new ones.
+        /// </summary>
+        /// <param name="data">Resources used for creating passes. In case of the null, no passes will be created.</param>
+        private void RecreatePostProcessPasses(PostProcessData data)
+        {
```

Comment:
Now we have same function in forward and 2d renderer. Maybe we should create base class that has this method? TBH I am not fan of inheritence, because very quikly we can end up with 5 levels of inheritence (This was huge problem in our old test framework, where inheritence is over-used).

Reply:
We already have ScriptableRenderer base class. You can move it there.

Reply:
K